charts/yurt-dashboard/templates/deployment.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "yurt-dashboard.fullname" . }}
+  labels:
+    {{- include "yurt-dashboard.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      {{- include "yurt-dashboard.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "yurt-dashboard.selectorLabels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ include "yurt-dashboard.fullname" . }}-backend
```

Comment:
Since yurt-dashboard is deployed in the cloud by default, how about adding the following tolerations here：
```
tolerations:
      - effect: NoSchedule
        key: node-role.openyurt.io/addon
        operator: Exists
      - effect: NoSchedule
        key: node-role.kubernetes.io/master
        operator: Exists
```

Reply:
Add  toleration key in chart, user can custom set them.